### DISCUSSION TITLE
Is it possible to setup  '.html' suffix on routes generated from /src/pages/xxxxx.md?

### DISCUSSION THREAD

**Discussion by jscheller** [antfu/vitesse#208] [Q&A]
Hello!

I'm trying to migrate a Nuxt.js static site project (several thousand markdown files) over to vite-ssg/vite-plugin-md/vite-plugin-pages using Vitesse as a starting point. I've made surprisingly rapid progress and I'm really excited to see how vite-ssg ultimately works with this.

For a number of reasons, I need to preserve the legacy URL structure and all of the generated routes currently end with an '.html' suffix on the end of whatever the markdown path was for the page. So for example, I need the router to respond to "https://domain.com/foo.html' for a file like /src/pages/foo.md.

I've been playing around with the routes array returned from setupLayouts() in main.ts by adding an alias or changing 'path' directly in there, but I can't seem to get anything that works.

Is there an easy way to accomplish this that I'm missing?

Any insights appreciated!

James

**Comment by hannoeru** [comment]
You can add alias to routes.

https://next.router.vuejs.org/guide/essentials/redirect-and-alias.html#alias
